crates/runtime/src/dataconnector/databricks.rs

```diff
@@ -111,7 +104,10 @@ impl Databricks {
                     }
                 };
             }
-            let Some(cluster_id) = ref_params.and_then(|p| p.get("databricks-cluster-id")) else {
+            let ((Some(cluster_id), _) | (_, Some(cluster_id))) = (
+                params.get("databricks_cluster_id"),
+                params.get("databricks-cluster-id"),
```

Comment:
I also changed this to accept either `databricks-cluster-id` or `databricks_cluster_id`. We can update the docs after the next release to use the underscore method.